src/In.ProjectEKA.OtpService/Notification/NotificationService.cs

```diff
@@ -0,0 +1,38 @@
+namespace In.ProjectEKA.OtpService.Notification
+{
+    using System.Threading.Tasks;
+    using Clients;
+    using Common;
+    using Newtonsoft.Json.Linq;
+
+    public class NotificationService : INotificationService
+    {
+        private readonly ISmsClient smsClient;
+
+        public NotificationService(ISmsClient smsClient)
+        {
+            this.smsClient = smsClient;
+        }
+
+        public async Task<Response> SendNotification(Notification notification)
+        {
+            return notification.Action switch
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace this 'switch' expression with a ternary conditional operator to increase readability.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5101780)